storage.js

```diff
@@ -0,0 +1,25 @@
+const ContactForm = document.querySelector('#footer_form');
+
+
+ContactForm.addEventListener('input', (e) => {
+  console.log("Change handler")
+  e.preventDefault();
```

Comment:
- [ Optional ]  Please it would be better to remove this console statement  to reduce the size of the javascript being sent to the client